Desktop cgxp interface route issue with 2.1.0RC1

Here the error, a route issue : 

https://preprod.cartoriviera.ch/v2/

Done, rebuiled, but still error 500  https://preprod.cartoriviera.ch/v2/

But still this error too during the build : 
Unable to DescribeFeatureType from URL /qgis/qgis_mapserver.fcgi?map=/var/sig/qgis/distribution.qgs&transparent=true&VERSION=1.1.0&REQUEST=DescribeFeatureType&SERVICE=WFS
No such table '' for layer 'separateurs_hydro'.
CONST_Makefile:686: recipe for target 'cartoriviera/locale/cartoriviera-client.pot' failed
make: *** [cartoriviera/locale/cartoriviera-client.pot] Error 1

--- a/c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl
+++ b/c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl
@@ -32,6 +32,20 @@ If you want to keep your CGXP interface you should:
   git mv {{package}}/templates/index.html {{package}}/templates/desktop.html
   git mv {{package}}/templates/viewer.js {{package}}/templates/desktop.js
 
+* In the `{{package}}/templates/desktop.html` file do the following replacement:
+
+    - <script type="text/javascript" src="${request.route_url('viewer', _query=extra_params)}"></script>
+    + <script type="text/javascript" src="${request.route_url('desktop.js', _query=extra_params)}"></script>
+
+* In the `{{package}}/templates/*.js` files remove the following line:
+
+    - OpenLayers.ProxyHost = "${request.route_url('ogcproxy') | n}?url=";
+
+* In the `jsbuild/app.cfg.mako` file do the following replacement:
+
+    - [app.js]
+    - [desktop.js]
+
 * Continue on the step 4., (you should do the step 1.-3. when you want
   to use the ngeo interface)
 

